installer/manifests/keptn/charts/control-plane/templates/api-service.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+            - name: AUTOMATIC_PROVISIONING_URL 
+              value: {{ .Values.apiService.features.automaticProvisioningURL | default "" }}
```

Comment:
this might lead to nil pointer exceptions when upgrading from a previous version with the `--reuse-values` flag. To prevent this, try to access this variable with:

```
{{ (.Values.apiService.features).automaticProvisioningURL | default "" }}
```